application/run-tests.sh

```diff
@@ -1,9 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
As a side note, in some projects I have ended up moving back to having a `run-tests.sh` file so that the way I run tests locally is the same as the way tests are run on CI/CD.